docker-compose-dev.yaml

```diff
@@ -277,6 +277,26 @@ services:
       - "./.dapr/dapr-config.yaml:/config.yaml"
       - "./.dapr/components:/components"
 
+  media:
+    extends:
+      file: media/docker-compose-base.yaml
+      service: media
+    ports:
+      - 8080:8080
```

Comment:
this port is already used by the gateway
also why expose any additional ports here anyway?

Reply:
Yeah, that ended up there while testing